eval/http.go

```diff
@@ -290,7 +292,45 @@ func ApplyResponseContext(ctx context.Context, body hcl.Body, beresp *http.Respo
 		return nil
 	}
 
-	return ApplyResponseHeaderOps(ctx, body, beresp.Header)
+	if err := ApplyResponseHeaderOps(ctx, body, beresp.Header); err != nil {
+		return err
+	}
+
+	content, _, _ := body.PartialContent(config.BackendInlineSchema)
+	if attr, ok := content.Attributes["set_response_status"]; ok {
+		var httpCtx *hcl.EvalContext
+		if c, ok := ctx.Value(ContextType).(*Context); ok {
+			httpCtx = c.eval
+		}
+
+		val, attrDiags := attr.Expr.Value(httpCtx)
+		if seetie.SetSeverityLevel(attrDiags).HasErrors() {
+			return attrDiags
+		}
+
+		status := seetie.ValueToInt(val)
+		if status < 100 || status > 599 {
+			return errors.Configuration.With(
```

Comment:
```suggestion
			return errors.Configuration.Label("set_response_status").Messagef("invalid http status code: %d", status)
```

Reply:
Additionally the user will see that this configuration error results in status-code `500`, as always.